src/domain/generator.ts

```diff
@@ -0,0 +1,55 @@
+import {GenerateTaskParams, Generator, Operator, Task} from "@/domain/domain";
+import {getRandomInteger} from "@/utils";
+
+const generator: Generator = {
+    generateTask(params: GenerateTaskParams): Task {
+        const {complexity, allowedOperators} = params;
+        const startValue = getRandomInteger(1, 10);
```

Comment:
```
class Generator {
  protected _MIN_ALLOWED_NUMBER = 1;

  public set MIN_ALLOWED_NUMBER(value: number) {
    if (value <= 0) throw ...
  }

 public get MIN_ALLOWED_NUMBER() {}

 /* гет-сет для максимального числа по аналогии */
}

Reply:
`const startValue = this.getRandomInteger(Generator.MIN_ALLOWED_NUMBER, GENERATOR.MAX_ALLOWED_NUMBER);`